src/CpmsClient/Service/CacheAwareApiService.php

```diff
@@ -1,87 +1,76 @@
 <?php
+
 namespace CpmsClient\Service;
 
+use Exception;
+use Laminas\Cache\Exception\ExceptionInterface;
 use Laminas\Cache\Storage\StorageInterface;
 use Laminas\Log\LoggerAwareTrait;
+use Laminas\Log\LoggerInterface;
 
 /**
  * Class ApiService
- * @method get
- * @method post
- * @method put
- * @method delete
  *
  * @package CpmsClient\Service
  */
 class CacheAwareApiService
 {
     use LoggerAwareTrait;
-    /**
-     * @var ApiService
-     */
-    protected $serviceProxy;
 
-    /** @var  StorageInterface */
-    protected $cacheStorage;
+    protected ApiService $serviceProxy;
+
+    protected StorageInterface $cacheStorage;
 
-    public function __construct(ApiService $service)
+    public function __construct(ApiService $service, LoggerInterface $logger, StorageInterface $cacheStorage)
```

Comment:
Can we use here "Constructor property promotion"?

Reply:
changed